Formula/podman.rb

```diff
@@ -0,0 +1,26 @@
+class Podman < Formula
+  desc "Tool for managing OCI containers and pods"
+  homepage "https://podman.io/"
+  url "https://github.com/containers/podman/archive/v2.0.5.tar.gz"
+  sha256 "cd7618429f02db8e7dbca29c6f8a7623a91a01804ef768dc5ec8c7660679b0c5"
+  license "Apache-2.0"
+
+  depends_on "go-md2man" => :build
+  depends_on "go@1.14" => :build
```

Comment:
Does the compilation fail with regular `go`?

Reply:
We use go 1.14 for our linux builds, so we wanted to match it here

Reply:
If it compiles with the latest version it must be used. Homebrew is an evergreen package manager after all.